src/pages/plan/utils/MemoDispatcherResolver.ts

```
@@ -0,0 +1,36 @@
+import { socketEventBus } from '../hooks/useSocketHandler';
+import type {
+  MemoCreateType,
+  MemoInitType,
+  MemoResponseType,
+  MemoUpdateType,
+} from '../types/MemoResponseBodyType';
+
+export function MemoDispatcherResolver(message: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)